### PR TITLE
Common sass

### PR DESCRIPTION
**Pull request by cupcakecoders**
added styles for the rover landing pages

### CODE REVIEW

File: src/Components/CuriosityHero/CuriosityHero.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿﻿import React from "react";
+import 'CuriosityHero.css';
```
Comment (MichaelWalker) [inline src/Components/CuriosityHero/CuriosityHero.tsx]: could you change `CuriosityHero` to be a `scss` file?

File: src/Components/Hero/Hero.scss [hunk 2]
```diff
@@ -12,12 +10,12 @@ $opportunityThemeColor: grey;
...
     align-items: center;
 
-    h1 {
+/*    h1 {
         width: 100%;
         text-align: center;
         color: white;
         font-size: 22px;
-    }
+    }*/
```
Comment (MichaelWalker) [inline src/Components/Hero/Hero.scss]: Rather than commenting it out, could this just be deleted?

**Review by MichaelWalker — CHANGES_REQUESTED** [review]
Two very tiny tweaks, but otherwise looks awesome!

File: src/Components/Hero/Hero.scss (hunk 2 above)
Comment (cupcakecoders) [reply src/Components/Hero/Hero.scss]: deleted

File: src/Components/CuriosityHero/CuriosityHero.tsx (hunk 1 above)
Comment (cupcakecoders) [reply src/Components/CuriosityHero/CuriosityHero.tsx]: changed

**Review by MichaelWalker — APPROVED** [review]

**Pull request merged**